code/_onclick/hud/screen_objects.dm

```diff
@@ -465,6 +467,17 @@
 
 	return TRUE
 
+/atom/movable/screen/storage/corner
+	name = "storage"
+	icon_state = "storage_corner_topleft"
+	screen_loc = "7,7 to 10,8"
+
+/atom/movable/screen/storage/rowjoin
+	name = "storage"
+	icon_state = "storage_rowjoin_left"
+	screen_loc = "7,7 to 7,7"
+	alpha = 0
```

Comment:
?

Reply:
By default rowjoins should be invisible since you'll only have one row, but since they need to be generated (and tiled) at least one of each will be present on the screen. This is just a solution to hide them when you only have one row.